client/web/compose/src/components/ModuleFields/Editor/File.vue

```diff
@@ -49,6 +64,62 @@
       mode="list"
       class="mt-2"
     />
+
+    <b-modal
+      ref="modal"
+      size="lg"
+      modal-class="video-modal-width"
+      :title="$t('editor.file.webcam.title')"
+      ok-only
+      ok-title="Close"
+      body-class="d-flex flex-column align-items-center"
+      @ok="closeCamera"
```

Comment:
Either pass events through the template #modal-footer (look at other components on how they do that)
Or remove the ok event.